src/DotNetProjectFile.Analyzers.Sdk/DotNetProjectFile.Analyzers.Sdk.props

```diff
@@ -32,7 +40,8 @@
     <AdditionalFiles Include="*.txt" />
     <AdditionalFiles Include="*.yaml" />
     <AdditionalFiles Include="*.yml" />
-    <AdditionalFiles Include="$(MSBuildProjectFile)" Visible="false" />
+    <AdditionalFiles Include="props/*.props" />
```

Comment:
Aren't these already included at line 25?

Reply:
Yes, but what is directly under the props folder, we want to see in the .net.csproj projects. The others we (just) need to make this rule work.